### ISSUE TITLE
Usage example

### ISSUE THREAD

**Issue by saitho** [l3lackShark/audio-share-discord-linux#7]
Can you provide an example on how to configure it properly?
As in, which output channel to select as I see 5 on my system.

I did manage to get it to working once. After I slightly adjusted the input volume in Discord it stopped working.

![inputs](https://user-images.githubusercontent.com/7293310/145676246-aaab1457-44da-4f9a-82e4-fe8f8e1f538d.png)

I _think_ I got it working by using [3] FunnelSink as output and outputting Spotify to "Do_Not_Touch_Sink" via pavucontrol.
However this is not how it's intended according to the existing instructions.

Also a documentation on how the three additional sound channels work would make understanding and debugging it easier I think.

**Comment by Amulek1416** [comment]
I agree. A small example in `README.md ` (or at least a link to it) would be helpful for those trying to use this.

**Comment by l3lackShark** [comment]
Sorry, it was not really meant to be used by this many people when I first made it, I'll consider a wiki
